Shared/UI/Views/MessageContentView/MessageContentView.swift

```diff
@@ -8,18 +8,8 @@
 
 import Foundation
 import Combine
-#if IOS
-import SwiftUI
-import StreamChat
-#endif
 
 class MessageContentView: BaseView {
-
-#if IOS
-    var handleTappedMessage: ((ConversationId, MessageId) -> Void)?
-    var handleEditMessage: ((ConversationId, MessageId) -> Void)?
-    var handleTappedAttachment: ((MediaItem) -> Void)?
```

Comment:
Removed all of the input handlers out of the content view. The content view is now just responsible for layout and view configuration.